src/main/webapp/jsp/instructorStudentRecords.jsp

```diff
@@ -205,6 +205,7 @@
                                 <div id="commentTextEdit<%=commentIdx%>" style="display:none;">
                                     <div class="form-group form-inline">
                                         <div class="form-group text-muted">
+                                            Comment about <%=InstructorStudentRecordsPageData.sanitizeForHtml(data.student.name)%>:<br>
```

Comment:
1 br is for comment about a student. 2 br is for comment about a feedback response. Used 2 br when there is a giver and a recipient